ts/features/wallet/bancomat/screen/BancomatDetailScreen.tsx

```diff
@@ -62,13 +62,13 @@ const UnsubscribeButton = (props: { onPress?: () => void }) => (
  */
 const startCoBadge = (props: Props) => {
   const bancomat = props.navigation.getParam("bancomat");
-  if (bancomat.abiInfo?.abi) {
-    props.addCoBadge(bancomat.abiInfo.abi);
-  } else {
+  if (bancomat.info.issuerAbiCode === undefined) {
     showToast(I18n.t("global.genericError"), "danger");
     void mixpanelTrack("BANCOMAT_DETAIL_NO_ABI_ERROR", {
       issuerAbiCode: bancomat.info.issuerAbiCode
```

Comment:
It will be undefined, so I guess this prop can be removed

Reply:
You're right! Removed in [9b66f5a](https://github.com/pagopa/io-app/pull/2940/commits/9b66f5a5658057b82638770ec1e45f3e27b0f3a2)